src/client/delphi_epidata.py

```diff
@@ -573,10 +573,13 @@ def covidcast(
       'signals': Epidata._list(signals),
       'time_type': time_type,
       'geo_type': geo_type,
-      'time_values': Epidata._list(time_values),
-      'geo_value': geo_value,
+      'time_values': Epidata._list(time_values)
     }
 
+    if isinstance(geo_value, (list, tuple)):
+      params['geo_values'] = ','.join(geo_value)
+    else:
+      params['geo_value'] = geo_value
```

Comment:
Same question -- why not `Epidata._list()`?

Reply:
in the current version changing it won't change much, since the array and plain version needs another parameter name `geo_value` vs `geo_values`. Thus, the `if` will remain.